templates/ovncontroller/bin/start-vswitchd.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+#
+# Copyright 2024 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+source $(dirname $0)/functions
+
+ovs_folder=/var/lib/openvswitch
+
+wait_for_ovsdb_server
+
+# Start vswitchd by asking it to wait till flow restore is finished.
+ovs-vsctl --no-wait set open_vswitch . other_config:flow-restore-wait=true
+/usr/sbin/ovs-vswitchd --pidfile --mlockall --detach
```

Comment:
I note that [ovs-ctl](https://github.com/openvswitch/ovs/blob/2c1a432e2f089f54c4aa395befbc6c2f07f0d305/utilities/ovs-ctl.in#L200-L232) does some additional things like setting MAXFDs and setting up logging. Do we need to do that here?

Reply:
@otherwiseguy maybe?.. Would be nice if you could check and report back. In the meantime, this PR does not attempt to change how the service is started. This command was used before to start the container, so there's no change here (except that --detach is added).